pipeline/builder/kubernetes.go

```diff
@@ -74,6 +74,7 @@ func (mp *ManifestParser) ContainersFromScaffold(scaffold config.ContainerScaffo
 	decode := scheme.Codecs.UniversalDeserializer().Decode
 	obj, g, err := decode(b, nil, nil)
 	if err != nil {
+		fmt.Printf("Marshaling Failure: %s\n", path)
```

Comment:
You should wrap this error instead and return that. For example:

```go
return nil, errors.Wrapf(err, "marshaling failure: %s", path)
```

Reply:
thank, TIL